src/shared/watchdog.c

```diff
@@ -462,20 +496,26 @@ static usec_t watchdog_calc_timeout(void) {
                 return watchdog_timeout;
 }
 
-usec_t watchdog_runtime_wait(void) {
+usec_t watchdog_runtime_wait(unsigned divisor) {
```

Comment:
maybe add `assert(divisor > 0);` here, to catch division by zero?

Reply:
It's called in exactly thee places, one of which is test code. I added the `/* divisor= */` comments as requested, so I *hope* nobody will change on of those integers to 0. This is not public API.